applications/challengedac.md

```diff
@@ -0,0 +1,109 @@
+
+## Team
+
+### Team members
+
+- **Team Leader:** Chace Eskam
+- **Co-founder:** Kent Eskam
+- **System Engineer:** Phung Ta
+
+### Legal Structure
+
+- **Registered Legal Entity:** CALI-PUMP LLC
+- **Registered Address:** 300 Carlsbad Village Dr, Carlsbad, CA 92008
+
+### Team Experience
+
+
+Chace Eskam: Has spent a number of years studying the social and economic impact of the blockchain. Chace has given talks about blockchains future implications on business at W.P Carey school of business as early as February 2016. Before devoting himself fully to blockchain Chace spent a number of years working in the fintech industry at the Nations first digital bank.
+
+### Team Org Repos
+
+- https://github.com/ChallengeDAC
+- https://github.com/ChallengeDAC/dacdart
+
+## Development Roadmap
+
+### Milestone Summary
+
+- **Total Estimated Duration:** 6 months 
+- **Full-Time Equivalent (FTE):** 1 FTE
+- **Total Costs:** 18,500 USD
+
+### Milestone 1 - Develop DacDart, a decentralized SDK, using EOS Node API to hook to ChallengeDac application blocks to view accounts, performs smart contracts
+
+- **Estimated duration:** 2 months
+- **FTE:**  1
+- **Costs:** 9,000 USD
+
+| ID  | Deliverable | Specification |
+| --- | ----------- | ------------- |
+| 1.  | Refactor EOS Node APIs | $7,000 is added to the development of refactoring the EOS Node APIs to seperate modules written in Dart |
+| 2.  | Quality Assurance and Documentation | $2,000 to test the code reliability and stability of the SDK in different platforms |
+
+### Milestone 2 - Launch the first marketing campaign to have the ChallengeDac community creates the quality EOS public organic contents
+
+- **Estimated duration:** 2 months
+- **FTE:**  1
+- **Costs:** 5,000 USD
+
+| ID  | Deliverable | Specification |
+| --- | ----------- | ------------- |
+| 1.  | Reward Pool Marketing Campaign No.1 | $2,000 is added to the reward pool for content creator and user support cost |
+| 2.  | Marketing Campaign No.1 | $3,000 is added to launch a campaign to have the ChallengeDac community creates the first 2,000 pieces of new EOS public organic contents |
```

Comment:
Also, how is the quality control of community created content is performed to determine reward applicability?

Reply:
Content is submitted first and able to be reviewed by the Challenge creator. The Challenge creator can choose what content gets rewarded and what content does not.
Additionally, there is an internal rating system for users to rank each other's credibility within the app.

Reply:
Thank you